### PR TITLE
New front page

### PR DESCRIPTION
**Pull request by logason**

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝 Changed routes:


| [**/index.module**](https://docs-6h3ustmqo.avo-dev.app/index.module) | [**/**](https://docs-6h3ustmqo.avo-dev.app/) |
|:-:|:-:|
|<a href="https://docs-6h3ustmqo.avo-dev.app/index.module">Screenshot of /index.module</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/YWfQqt1xoXmTr80mbys0BFAziliYBVr8vBIzqU8ql-BNP5xq63cKeu5dj0bZW_1i16aH-F-aj7SP9l2MnFBX3w.png">(view full size)</a> |<a href="https://docs-6h3ustmqo.avo-dev.app/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/Gz253_UUBN2UknhHBJDcYh_LvJOFiZdjJCRmW8o_p3J_uVNemSJKHF7gcDKK5pLm_Rol7KEz9qHJVQ2wza4sHg.png">(view full size)</a> |







Commit 7125ede00023221d394ac13bffa8b2dce1c13246 (https://docs-6h3ustmqo.avo-dev.app).

File: components/Link.tsx [hunk 1]
```diff
@@ -4,13 +4,13 @@ import getConfig from 'next/config';
 
 const { publicRuntimeConfig } = getConfig();
 
-const Link: React.FunctionComponent<LinkProps> = ({ children, ...props }) => (
-  <NextLink
+const Link: React.FunctionComponent<LinkProps> = ({ children, ...props }) => {
+  return (<NextLink
```
Comment (hugihlynsson) [inline components/Link.tsx:8]: Hmm, is this better?

File: package.json [hunk 2]
```diff
@@ -40,6 +40,7 @@
     "react-dom": "^17.0.0",
     "react-hamburger-button": "^0.0.1",
     "react-innertext": "^1.1.5",
+    "react-medium-image-zoom": "4.4.3",
```
Comment (hugihlynsson) [inline package.json:43]: 👌

File: pages/index.tsx [hunk 3]
```diff
@@ -21,555 +24,282 @@ const LandingPage: FunctionComponent = () => {
...
+          release.
         </p>
-        <h3 className={[styles.label, styles.heading3].join(' ')}>
-          Creating tracking plan in Avo
-        </h3>
+        <p className={styles.callout}>
+          <span>💡</span>
+          When trying the Avo Workflow for the first time, don't choose a
+          product release that's about to take the next 3 months to work on.
+          Choose one that has a handful of new or updated events.
```
Comment (hugihlynsson) [inline pages/index.tsx:93]: Should we frame this in a more positive way? Something in this directly perhaps:
```suggestion
          When trying the Avo Workflow for the first time, choose a
          product release that has a handful of new or updated events, rather 
          than something that takes the next 3 montes to work on.
```

File: pages/index.tsx [hunk 4]
```diff
@@ -21,555 +24,282 @@ const LandingPage: FunctionComponent = () => {
...
+        </Zoom>
+
         <p className={styles.text}>
-          If you already have a tracking plan you can import it to Avo, or you
-          can create your plan in Avo from scratch. In any case you&apos;ll be
-          able to fine tune it and add new events for every feature release in
-          Avo.
+          We recommend your first “full-circle” release in Avo be a small one.
+          This will enable you to quickly go through your first analytics
+          release in Avo, from start to finish.
```
Comment (hugihlynsson) [inline pages/index.tsx:108]: Do we want to repeat the callout content here?

**Review by hugihlynsson — COMMENTED** [review]
Overall it looks good!

File: components/Link.tsx (hunk 1 above)
Comment (logason) [reply components/Link.tsx]: ahh no just some debugging that I forgot to fully revert, fixed in ba32008f3b69336d83c8f207795f95c52796f753

File: pages/index.tsx (hunk 4 above)
Comment (logason) [reply pages/index.tsx]: Merge some of the copy from the callout above with this one in ba32008f3b69336d83c8f207795f95c52796f753, let me know what you think @hugihlynsson!

**Review by hugihlynsson — APPROVED** [review]
LGTM

**Pull request merged**